freeipa/src/main/resources/freeipa-salt/salt/freeipa/common-install.sls

```diff
@@ -39,4 +39,53 @@ restart_krb5kdc:
   service.running:
     - name: krb5kdc
     - watch:
-      - file: /etc/sysconfig/krb5kdc
\ No newline at end of file
+      - file: /etc/sysconfig/krb5kdc
+
+/usr/local/bin/freeipa_backup:
+  file.managed:
+    - source: salt://freeipa/scripts/freeipa_backup
+    - user: root
+    - group: root
+    - mode: 750
+
+{% if salt['pillar.get']('freeipa:backup_enabled') %}
+/etc/freeipa_backup.conf:
+  file.managed:
+    - source: salt://freeipa/templates/freeipa_backup.conf.j2
+    - template: jinja
+    - user: root
+    - group: root
+    - mode: 640
+
+{% if salt['pillar.get']('freeipa:monthly_full_backup_enabled') %}
+/etc/cron.monthly/freeipa_backup_monthly:
+  file.managed:
```

Comment:
No because it is not using cron it is using anacron.  This is necessary because anacron puts in some randomization of runtime for parts like monthly.  We don't want all the monthly backups to kick off at the same time in an HA environment because the monthly full backup brings down the freeipa for about 2 minutes.